modules/agent/http/plugin.go

```diff
@@ -2,13 +2,36 @@ package http
 
 import (
 	"fmt"
+	"net/http"
+	"os/exec"
+
 	"github.com/Cepave/open-falcon-backend/modules/agent/g"
 	"github.com/Cepave/open-falcon-backend/modules/agent/plugins"
 	"github.com/toolkits/file"
-	"net/http"
-	"os/exec"
 )
 
+func reCloneGit(w http.ResponseWriter) {
+	dir := g.Config().Plugin.Dir
+	parentDir := file.Dir(dir)
+	cmd1 := exec.Command("rm", "-rf", file.Basename(dir))
```

Comment:
Do some check here to avoid `rm -rf /`. (e.g., whether Prefix of directory is `/usr/local/owl` or not)